### PR TITLE
store comments in Datastore

### PR DESCRIPTION
**Pull request by Andrewmob2703-zz**

### CODE REVIEW

**Comment by Andrewmob2703-zz** [comment]
I can see my comments in the local bin file but I don't see them on the admin page, am I doing sth wrong?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -28,18 +31,30 @@
...
     response.setContentType("text/html;");
 
-    if (getMessage.equals("")) {
+    if (comment.equals("")) {
       finalResponse = "Got no comment.";
       response.getWriter().println("<h3>" + finalResponse + "</h3>");
-    }else {
-      finalResponse = "Submitted! Thank you " + getParameter(request, NAME, "");
+    } else {
+      finalResponse = "Submitted! Thank you " + user;
```
Comment (peizhaoou) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:55]: Nit: when user is equal to empty string, the returned message will end with an extra space. This is no big deal for now, but good to keep corner cases in mind.

**Review by peizhaoou — APPROVED** [review]
LGTM.

Re >> "I can see my comments in the local bin file but I don't see them on the admin page, am I doing sth wrong?"

Not sure where to respond to your question so I will add something here. I may need more context, can you elaborate your question a bit? Do you mean the comments posted by the user stored in a local bin file. I haven't used datastore myself so I can be wrong, but I assume it's expected that comments are stored in the hard drive.

What do you mean by seeing them in the admin page?

**Comment by Andrewmob2703-zz** [comment]
> LGTM.
> 
> Re >> "I can see my comments in the local bin file but I don't see them on the admin page, am I doing sth wrong?"
> 
> Not sure where to respond to your question so I will add something here. I may need more context, can you elaborate your question a bit? Do you mean the comments posted by the user stored in a local bin file. I haven't used datastore myself so I can be wrong, but I assume it's expected that comments are stored in the hard drive.
> 
> What do you mean by seeing them in the admin page?

There's a link to the Datastore admin page, but I'm not sure if we're supposed to see the comments there, but I think the tutorial says so. I might have to ask a few people for confirmation.

**Pull request merged**